Makefile

```diff
@@ -14,36 +14,38 @@ BUILD_META=-build$(shell date +%Y%m%d)
 ORG ?= rancher
 TAG ?= v4.0.2$(BUILD_META)
 
-ifneq ($(DRONE_TAG),)
-	TAG := $(DRONE_TAG)
-endif
-
 ifeq (,$(filter %$(BUILD_META),$(TAG)))
-	$(error TAG needs to end with build metadata: $(BUILD_META))
+	$(error TAG $(TAG) needs to end with build metadata: $(BUILD_META))
```

Comment:
I think this needs to be moved to the left, otherwise Makefile thinks it's a target

Reply:
all the Makefiles are written like this, does it create problems?

Reply:
Yes. I changed it in a couple of them but not in all. Just try this: `TAG=whatever make`

Reply:
OK. I read 'move right' instead of 'move left'...